tests/defaults_list/test_defaults_tree.py

```diff
@@ -1302,6 +1302,31 @@ def test_extension_use_cases(
             ),
             id="with_missing:override",
         ),
+        param(
+            "with_missing_at_foo",
+            [],
+            raises(
+                ConfigCompositionException,
+                match=dedent(
+                    """\
+                You must specify 'db@foo', e.g, db@foo=<OPTION>
+                Available options:"""
+                ),
+            ),
+            id="with_missing_at_foo",
+        ),
+        param(
+            "with_missing_at_foo",
+            ["db@foo=base_db"],
+            DefaultsTreeNode(
+                node=ConfigDefault(path="with_missing_at_foo"),
+                children=[
+                    GroupDefault(group="db", value="base_db", package="foo"),
+                    ConfigDefault(path="_self_"),
+                ],
+            ),
+            id="with_missing_at_foo:override",
```

Comment:
Is this not already tested?

Reply:
No, these tests are failing on the master branch.